feil-behandler/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/feilbehandler/river/FeilLytterTest.kt

```diff
@@ -47,23 +53,57 @@ class FeilLytterTest : FunSpec({
     }
 
     test("skal ignorere feil uten forespørselId") {
+        // TODO: Kan egentlig tillate feil uten forespørselId..
         val feil = lagGyldigFeil(BehovType.JOURNALFOER).copy(forespoerselId = null)
         handler.skalHaandteres(feil) shouldBe false
     }
 
-    test("skal ignorere ugyldige feil") {
-        val feil = toFailOrNull(emptyMap())
-        handler.skalHaandteres(feil) shouldBe false
+    test("Ny feil med forskjellig behov og samme id skal lagres, gjentakende feil oppdaterer jobb") {
```

Comment:
> gjentakende feil oppdaterer jobb

Testes dette her?

Reply:
Nei, for det ble flyttet til neste test! Endrer teksten..